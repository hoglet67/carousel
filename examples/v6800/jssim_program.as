        ORG $8000

TOP:        
        LDAA #0
        STAA $0F
        LDX #$4321
        TXS
LOOP:
        LDAB #$FB
        JSR L2
        JMP LOOP
        NOP
        NOP
        NOP
        NOP
L2:
        INX
        DECA
        INC $0F
        SEC
        ADCB #$02
        RTS

