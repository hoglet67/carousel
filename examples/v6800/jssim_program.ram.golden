  
0000: 00 ff ff ff ff ff ff ff ff ff ff ff ff ff ff 3a   ...............:
0010: ff ff ff ff ff ff ff ff ff ff ff ff ff ff ff ff   ................
0020: ff ff ff ff ff ff ff ff ff ff ff ff ff ff ff ff   ................
0030: ff ff ff ff ff ff ff ff ff ff ff ff ff ff ff ff   ................
0040: ff ff ff ff ff ff ff ff ff ff ff ff ff ff ff ff   ................
0050: ff ff ff ff ff ff ff ff ff ff ff ff ff ff ff ff   ................
0060: ff ff ff ff ff ff ff ff ff ff ff ff ff ff ff ff   ................
0070: ff ff ff ff ff ff ff ff ff ff ff ff ff ff ff ff   ................
0080: ff ff ff ff ff ff ff ff ff ff ff ff ff ff ff ff   ................
0090: ff ff ff ff ff ff ff ff ff ff ff ff ff ff ff ff   ................
00a0: ff ff ff ff ff ff ff ff ff ff ff ff ff ff ff ff   ................
00b0: ff ff ff ff ff ff ff ff ff ff ff ff ff ff ff ff   ................
00c0: ff ff ff ff ff ff ff ff ff ff ff ff ff ff ff ff   ................
00d0: ff ff ff ff ff ff ff ff ff ff ff ff ff ff ff ff   ................
00e0: ff ff ff ff ff ff ff ff ff ff ff ff ff ff ff ff   ................
00f0: ff ff ff ff ff ff ff ff ff ff ff ff ff ff ff ff   ................
